dask_cuda/tests/test_local_cuda_cluster.py

```diff
@@ -350,7 +355,10 @@ async def test_gpu_uuid():
 async def test_rmm_track_allocations():
     rmm = pytest.importorskip("rmm")
     async with LocalCUDACluster(
-        rmm_pool_size="2GB", asynchronous=True, rmm_track_allocations=True
+        rmm_pool_size="2GB",
+        asynchronous=True,
+        rmm_track_allocations=True,
+        device_memory_limit="30B",
```

Comment:
Why this change here?

Reply:
This was an accident while I was mucking about with config tests.